robot/Cumulus/tests/browser/recurring_donations/zcreate_open_erd_for_contact.robot

```diff
@@ -0,0 +1,60 @@
+*** Settings ***
+Resource        robot/Cumulus/resources/NPSP.robot
+Library         cumulusci.robotframework.PageObjects
+...             robot/Cumulus/resources/RecurringDonationsPageObject.py
+...             robot/Cumulus/resources/NPSP.py
+Suite Setup     Run keywords
+...             Open Test Browser
+...             Setup Test Data
+...             Enable RD2
+Suite Teardown  Delete Records and Close Browser
+
+
+***Keywords***
+# Setup a contact with parameters specified
+Setup Test Data
+    Setupdata           contact               contact_data=${contact_fields}
```

Comment:
How about adding that comment as a Documentation for the keyword? In the future, when robot_lint task becomes mandatory, this will raise an error.